ARViewer/ViewController.swift

```diff
@@ -116,6 +119,19 @@ class ViewController: UIViewController, ARSCNViewDelegate, SCNPhysicsContactDele
     // MARK: - Actions
     
     @IBAction func didTapScreen(_ sender: UITapGestureRecognizer) { // fire bullet in direction camera is facing
+        
+        do
+        {
+            let MusicURL = Bundle.main.path(forResource: "torpedo", ofType: "mp3")
+            try Player = AVAudioPlayer(contentsOf: NSURL (fileURLWithPath:MusicURL!) as URL)
+            Player.prepareToPlay()
```

Comment:
prepareToPlay() doesn't have adequate separation from Play() instruction to be useful